kubeflow/openmpi/assets.libsonnet

```diff
@@ -0,0 +1,128 @@
+{
+  all(params):: {
+    kind: "ConfigMap",
+    apiVersion: "v1",
+    metadata: {
+      name: "openmpi-assets",
+      labels: {
+        app: params.name,
+      },
+    },
+    data: {
+      "gen_hostfile.sh": |||
```

Comment:
I think its more readable to put the scripts in files e.g. gen_hostfile.sh    and then do
importstr "gen_hostfile",

Reply:
Thanks for the suggestion. I have moved all scripts into the assets directory.